keyboards/keychron/q1/rev_0100/keymaps/teimor/keymap.c

```diff
@@ -0,0 +1,116 @@
+/* Copyright 2022 @ Teimor Epstein
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "keymap_user.h"
+#ifdef RGB_MATRIX_ENABLE
+#    include "rgb_matrix_user.h"
+#endif
+#include "tap_dance_user.c"
```

Comment:
c files should not be included like this. 
```suggestion
```

It should be added via `SRC += tap_dance_user.c` in the rules.mk file for your keymap.